pkgs/by-name/or/orca-slicer/package.nix

```diff
@@ -39,6 +39,8 @@
   wxGTK31,
   xorg,
   withSystemd ? stdenv.hostPlatform.isLinux,
+  # By default, orca-slicer automatically checks if a new version is available on startup, presumably so that users remember to update. This doesn't really apply when orca-slicer is obtained using nix, so disable it by default in the nix package
+  doVersionCheck ? false,
```

Comment:
```suggestion
```
we absolutely do not require a setting for this. We patch this unconditionally out of many packages.

Reply:
This option does not make much sense. Especially because the update mechanism is broken and doesn't work anyway.